src/utils/pathUtils.ts

```diff
@@ -18,3 +19,12 @@ export function normalisePath(path: string, withPrefix?: string, withSuffix?: st
   normalised = normalised.replace(/[/\\\s]+/g, '/');
   return normalised;
 }
+
+export function convertColonPathParams(path: string) {
```

Comment:
minor nitpick: I think it's reasonable to do this inside `normalizePath`

Reply:
I agree and I put it there at first, but it made route generation tests fail.

Reply:
Yes, we are calling this again while building the templates. I assume this is the better approach then.